### DISCUSSION TITLE
Apk size

### DISCUSSION THREAD

**Discussion by tomaszk8266** [IacobIonut01/Gallery#453] [Q&A] [answered]
For some reason, the APK size growed up to 132 MB (it was previously about 30 MB). What is it caused by?

**Comment by IacobIonut01** [comment, marked as answer]
It contains all the libraries used to decode specific image formats like avif, jxl etc for multiple abis

**Comment by tomaszk8266** [comment]
There is apk splitting, AFAIK fdroid supports it, why don't use it?

**Comment by IacobIonut01** [comment]
f-droid and gplay are already distributing splitted apks, I usually upload just the universal one, but I may change that to upload all variants incl universal

**Reply by k2helix** [reply]
would be greatly appreciated

**Reply by namesarebloat** [reply]
>f-droid and gplay are already distributing splitted apks

It doesn't seem like fdroid is doing that. I think the issue may be that [they need different version codes per ABI](https://gitlab.com/fdroid/fdroiddata/-/issues/2809)?

The package on the play store is split though, aurora store says size is 35mb.

**Comment by serrq** [comment]
In my case I just need of the v8a one.

**Comment by rafnov** [comment]
I don't think I would ever need some exotic formats I never heard of (or maybe heard, but ignored as not needed), so what's the point of keeping 100 MB libs on me device, while gallery alone was <30MB?
